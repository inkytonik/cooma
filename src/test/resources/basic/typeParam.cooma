{
    val id = fun (t : Type, x : t) x
    {i = id(Int, 10), s = id(String, "hello")}
}
