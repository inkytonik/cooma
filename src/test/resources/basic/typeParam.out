{ i = 10, s = "hello" }
